app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/image_with_borders.phtml

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
```

Comment:
Remove Magento copyrights please, and add template annotation about changes

Reply:
File for test. Deleted